components/Advocates/AdvocatesItemCard.vue

```diff
@@ -4,6 +4,7 @@
     :image="image"
     :title="name"
     :tags="formattedRegion"
+    :image-description="name + ` photo`"
```

Comment:
Since you're using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), you can do this:

```suggestion
    :image-description="`${ name } photo`"
```